### ISSUE TITLE
Our app still needs to support version 5 : Can version 5.23 be used or I should stick to version 5.22 ?

### ISSUE THREAD

**Issue by omasseau** [Activiti/Activiti#4310]
Hello,

Our application still needs to support Activiti 5, until we plan migrating to the latest version.
We still want to be on the latest patch.
Is version 5.23 official and can be used without problem ? I'm wondering as there is no release notes, and I can see a lot of changes when comparing it with 5.22 : 
https://github.com/Activiti/Activiti/releases/tag/5.23.0 

Or is it recommended to stick to version 5.22, which seems the latest official release (as displayed in the User guide) ?
![image](https://user-images.githubusercontent.com/11772429/234235019-6ea60ca9-69a6-4834-9dd9-5b775fba7043.png)

In short, which version is recommended if still running Activiti 5 : 5.22 or 5.23 ?
